modules/ROOT/pages/to-configure-api-settings.adoc

```diff
@@ -16,6 +16,18 @@ In a private Exchange instance, asset administrators have access to change the v
 
 REST APIs provide a mocking service with data to test API features. The mocking service is always public.
 
+== Configure Non-Managed Instances 
+Exchange enables you to configure APIs with instances not managed by API Manager. Add, edit, and delete non-managed instances that are serving AsyncAPIs, GraphQL, REST, and HTTP APIs. You can change the name, URL, and visibility of a non-managed instances.
+
```

Comment:
Is this three separate tasks? Add, edit, and delete? If so, suggest separating them into different procedures so it doesn't imply you do all the steps.

You could use a bullet or definition list:

* Add an instance:
   a. blah
* Edit...
   a. blah

Reply:
good idea